test/channel_merge_zip_with_tests.cpp

```diff
@@ -235,7 +235,7 @@ BOOST_FIXTURE_TEST_CASE(int_zip_width_channel_different_type_void_functor,
     std::atomic_int result{0};
 
     auto check = zip_with(default_executor,
-                      [&](int, std::string y) { result += 2 + static_cast<int>(y.size()); },
```

Comment:
A call by const& is many cases the typical choice. But within futures and channels everything that can be moved is been moved. And I see the arguments of a channel-process as sink argument, so I have written the tests with that in mind. Is there any particular reason for this request?

Reply:
I do not remember why I changed this.
You are right, I will close the PR.
Sorry for the noise.